### PR TITLE
2226 detect boot state

### PR DESCRIPTION
**Pull request by jimmykarily**
Part of https://github.com/kairos-io/kairos/issues/2226

### CODE REVIEW

File: pkg/utils/common.go [hunk 1]
```diff
@@ -59,7 +60,7 @@ func GetUkiCmdline() []string {
 	cmdlineOverride := viper.GetStringSlice("cmdline")
 	if len(cmdlineOverride) == 0 {
 		// For no extra cmdline, we default to install mode
-		return []string{constants.UkiCmdline + " " + constants.UkiCmdlineInstall}
+		return []string{constants.UkiCmdlineInstall + " " + constants.UkiCmdline}
```
Comment (mauromorales) [inline pkg/utils/common.go:63]: I think this might have been why we were seeing the weird names :P

Comment (jimmykarily) [reply pkg/utils/common.go:63]: I don't recall doing this :D. Let me revert it.

Comment (jimmykarily) [reply pkg/utils/common.go]: image
I'm not sure if that made any change

Comment (mauromorales) [reply pkg/utils/common.go]: weeeeird

**Pull request merged**